process/sandbox-annual-review.md

```diff
@@ -0,0 +1,43 @@
+# Sandbox annual review 
+
+Sandbox projects are subject to an annual review by the TOC. This is intended to be a lightweight process to ensure that projects are on track, and getting the support they need.
```

Comment:
If there are SIGs in place, will the annual process first go through the SIG or will it directly be submitted to TOC?

Reply:
I think this should be a single-pase process to keep it simple and lightweight. Will clarify.